dashboard/templates/base.html

```diff
@@ -36,16 +31,13 @@
         };
     </script>
 </head>
-<body class="container-fluid">
+<body>
     {% include "su/is_su.html" %}
     {% block content %}
     {% endblock %}
     <footer>
-        <hr>
-        <div class="row">
-            <div class="col text-left">Copyright © 2018 The Regents of the University of Michigan</div>
-            <div class="col text-right">Data last updated on {{last_updated|date:"m/d/Y P T"}}</div>
-        </div>
+        <div style="float: left; width: 50%">Copyright © 2018 The Regents of the University of Michigan</div>
```

Comment:
Fixing this was part of #523 but it's fine to leave it as it is for now.

Reply:
Right, this should come from the env variables.